collections.go

```diff
@@ -77,8 +77,10 @@ type CollectionDTO struct {
 	ID             int64     `json:"id"`
 	PublishedAt    time.Time `json:"published_at"`
 	PublishedScope string    `json:"published_scope"`
+	Rules          RuleDTOs  `json:"rules"`
```

Comment:
again this only exists on smart collections, not collections, you need a completely separate set of types and repos

Reply:
Yeah I did see, but rules does actually come out of this api and surely we can enhance our shopify package. because in some cases it is like the docs in out of date.